pkg/validator/validators.go

```diff
@@ -266,3 +268,34 @@ func (kc *policyValidator) upstreamTrafficSettingValidator(req *admissionv1.Admi
 
 	return nil, nil
 }
+
+func meshRootCertificateValidator(req *admissionv1.AdmissionRequest) (*admissionv1.AdmissionResponse, error) {
+	switch req.Operation {
+	case admissionv1.Create:
```

Comment:
do we want to try to verify the state outside of the existing MRC, and that the system can handle the creation of a new one? Maybe add a comment on whichever we choose

Reply:
Will address this in a follow up PR